packages/web-components/fast-foundation/custom-elements-manifest-plugins.mjs

```diff
@@ -0,0 +1,141 @@
+import { getClassMemberDoc } from "@custom-elements-manifest/analyzer/src/utils/manifest-helpers.js";
+import { parse } from "comment-parser";
+
+/**
```

Comment:
FYI fast-foundation is a module package now, so you should be able to just use the `.js` extension if that is preferred.

Reply:
I'm going to need to update some of this once we upgrade the API-Extractor and I add support for @slot tags. I can change the file extensions then.